### PR TITLE
[COMMITTERS] Add new committers

### PR DESCRIPTION
**Pull request by mundaym**
Adding Miles and Marno to the list of committers. Thanks for all your contributions!

### CODE REVIEW

**Review by GregAC — APPROVED** [review]

**Review by marnovandermaas — COMMENTED** [review]
Thanks, happy to be a committer. I'll let others approve the PR because it seems weird to approve my own committer status.

**Pull request merged**
